src/metabase/segments/models/segment.clj

```diff
@@ -30,8 +34,34 @@
 (methodical/defmethod t2/table-name :model/Segment [_model] :segment)
 (methodical/defmethod t2/model-for-automagic-hydration [:default :segment] [_original-model _k] :model/Segment)
 
+(mr/def ::segment-definition
```

Comment:
this was previously shared between (Legacy) Metric and Segment and lived in a shared namespace; moved here since it's only used by Segments now.